src/tv2-common/migrations/index.ts

```diff
@@ -171,15 +172,76 @@ export function mapGfxTemplateToDesignTemplateAndDeleteOriginals(
 	})
 }
 
+export function moveSelectedGfxSetupNameToGfxDefaults(versionStr: string) {
+	const fromValue = 'SelectedGfxSetupName'
+	const targetTable = 'GfxDefaults'
+	return literal<MigrationStepShowStyle>({
+		id: `${versionStr}.moveSelectedGfxSetupName.${fromValue}.toTable.${targetTable}`,
+		version: versionStr,
+		canBeRunAutomatically: true,
+		validate: (context: MigrationContextShowStyle) => {
+			const singleValue = context.getBaseConfig(fromValue)
+			const designatedTable = context.getBaseConfig(targetTable)
+
+			if (!singleValue || !Array.isArray(designatedTable)) {
+				return false
+			}
+
+			return true
+		},
+		migrate: (context: MigrationContextShowStyle) => {
+			const singleValue = context.getBaseConfig(fromValue) as BasicConfigItemValue
+			const designatedTable = context.getBaseConfig(targetTable) as unknown as TableConfigItemValue
+			const setupName = 'DefaultSetupName'
+
+			designatedTable[0][setupName] = singleValue
+
+			context.setBaseConfig(targetTable, designatedTable)
+			context.removeBaseConfig(fromValue)
+		}
+	})
+}
+
+export function moveSelectedGfxSetupNameToGfxDefaultsInVariants(migrationVersion: string) {
+	const fromValue = 'SelectedGfxSetupName'
+	const targetTable = 'GfxDefaults'
+	return literal<MigrationStepShowStyle>({
+		id: `${migrationVersion}.moveSelectedGfxSetupNameToGfxDefaultsInVariants.${fromValue}.toTable.${targetTable}`,
+		version: migrationVersion,
+		canBeRunAutomatically: true,
+		validate: (context: MigrationContextShowStyle) => {
+			const allVariants = context.getAllVariants()
+
+			return allVariants.some((variant: IBlueprintShowStyleVariant) => {
+				return context.getVariantConfig(variant._id, fromValue) && !context.getVariantConfig(variant._id, targetTable)
+			})
+		},
+		migrate: (context: MigrationContextShowStyle) => {
+			const allVariants = context.getAllVariants()
+			allVariants.forEach((variant: IBlueprintShowStyleVariant) => {
+				const singleValue = context.getVariantConfig(variant._id, fromValue) as BasicConfigItemValue
+				let designatedTable = context.getVariantConfig(variant._id, targetTable) as TableConfigItemValue
+				const setupName = 'DefaultSetupName'
+
+				if (designatedTable === undefined) {
+					designatedTable = [{ [setupName]: singleValue, _id: '' }]
+				}
```

Comment:
If you use the fallback value with nullish coalescing, you can condense the this to:
```typescript
const setupName = 'DefaultSetupName'
const singleValue = context.getVariantConfig(variant._id, fromValue)
const designatedTable = context.getVariantConfig(variant._id, targetTable) ?? [{ [setupName]: singleValue, _id: '' }]
```
I have removed the type casting in the example – are they necessary?